css/style.css

```diff
@@ -0,0 +1,364 @@
+body{
```

Comment:
Common style approach - using space between tag and '{' and also space before ':' ( body{ => body { ). Pease, check it and fix in whole file.